aws/resource_aws_service_discovery_service.go

```diff
@@ -55,11 +55,17 @@ func resourceAwsServiceDiscoveryService() *schema.Resource {
 										Type:         schema.TypeString,
 										Required:     true,
 										ForceNew:     true,
-										ValidateFunc: validateServiceDiscoveryServiceDnsRecordsType,
+										ValidateFunc: validateStringIn("SRV", "A", "AAAA", "CNAME"),
 									},
 								},
 							},
 						},
+						"routing_policy": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ForceNew:     true,
+							ValidateFunc: validateStringIn("MULTIVALUE", "WEIGHTED"),
+						},
```

Comment:
Since the API defaults to `MULTIVALUE` shouldn't we also set it as `Default` here to avoid diffs when the field is not defined?